packages/i18n/src/utils/key.ts

```diff
@@ -8,9 +8,14 @@ import type { I18nBundle, I18nEntry } from './../types';
  * @returns extracted key
  */
 export function extractI18nKey(input: string, key = 'i18n'): string {
-    input = input.trim();
-    const regPattern = new RegExp(`^({@?${key}(>|&gt;))`, 'g');
-    return input.replace(regPattern, '').replace(/\}$/gm, '').trim();
+    let sanitizedInput = input.trim();
+
+    if (doesDoubleCurlyBracketsExist(sanitizedInput)) {
+        sanitizedInput = '{' + (extractDoubleCurlyBracketsKey(sanitizedInput) ?? '') + '}';
+    }
```

Comment:
Currently this function was only used for UI5 binding expressions and I don't think we should mix these two use cases, because they are context dependent. In some cases only one of the syntax variants are valid and we should check that.